### PR TITLE
Issue templates

### LINKED ISSUE #1591: Create an issue template

### PR DESCRIPTION
**Pull request by alex-semenyuk**
Close #1591

Based on https://github.com/hyperledger-labs/documentation-template/tree/main/.github/ISSUE_TEMPLATE

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Support Request
+    url: https://discord.gg/hyperledger
```
Comment (EnriqueL8) [inline .github/ISSUE_TEMPLATE/config.yml:4]: This needs to be updated to the LFDT discord URL

**Review by EnriqueL8 — CHANGES_REQUESTED** [review]
Looks good! Thanks for raising it, just one small change needed

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        - Use English (EN/US) to communicate.
+        - Search the [open issues](https://github.com/hyperledger/firefly/issues) to avoid duplicating the issue.
+
+  - type: textarea
+    id: problem
+    attributes:
+      label: What happened?
+      description: |
+        Please provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner.
+        If this matter is security related, please disclose it privately via security@hyperledger.org
```
Comment (EnriqueL8) [inline .github/ISSUE_TEMPLATE/bug-report.yml:20]: This email might also be different - @tkuhrt @ryjones fyi

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (EnriqueL8) [reply .github/ISSUE_TEMPLATE/config.yml:4]: Actually just validated that is work! So just the email below that needs changing maybe

Comment (ryjones) [reply .github/ISSUE_TEMPLATE/config.yml:4]: we can't change that link for boring reasons

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 2 above)
Comment (ryjones) [reply .github/ISSUE_TEMPLATE/bug-report.yml:20]: security@lfdecentralizedtrust.org

Comment (alex-semenyuk) [reply .github/ISSUE_TEMPLATE/bug-report.yml]: Thanks, updated

**Review by EnriqueL8 — APPROVED** [review]
Looks good

**Pull request merged**